nerfstudio/data/dataparsers/base_dataparser.py

```diff
@@ -60,6 +60,17 @@ class DataparserOutputs:
     """Scene box of dataset. Used to bound the scene or provide the scene scale depending on model."""
     mask_filenames: Optional[List[Path]] = None
     """Filenames for any masks that are required"""
+    depths: Optional[List[torch.Tensor]] = None
+    """Monocular depth."""
+    normals: Optional[List[torch.Tensor]] = None
+    """Monocular normal."""
```

Comment:
Can these be included in metadata?

Reply:
see comment above, if we include these in metadata, we have to generate a new dataset rather then using the base_dataset.

Reply:
I think it makes sense to create a new dataset in this case since it contains different info.